openmetadata-ui/src/main/resources/ui/src/assets/svg/ic-delete.svg

```diff
@@ -1,3 +1,5 @@
-<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24">
-<path fill="currentColor" d="M20.184 2.813H16.67v-.704C16.669.946 15.722 0 14.559 0H8.934C7.771 0 6.825.946 6.825 2.11v.703H3.309c-1.163 0-2.109.946-2.109 2.109 0 .934.61 1.728 1.453 2.004l1.254 15.14A2.122 2.122 0 0 0 6.01 24h11.474c1.089 0 2.012-.85 2.102-1.935L20.84 6.926a2.113 2.113 0 0 0 1.454-2.004c0-1.163-.947-2.11-2.11-2.11ZM8.231 2.109c0-.387.316-.703.703-.703h5.625c.388 0 .704.316.704.703v.704H8.23v-.704Zm9.954 19.84a.707.707 0 0 1-.7.645H6.01a.707.707 0 0 1-.701-.645L4.073 7.031h15.348L18.184 21.95Zm2-16.324H3.308a.704.704 0 0 1 0-1.406h16.875a.704.704 0 0 1 0 1.406Z"/><path fill="currentColor" d="M9.186 20.44 8.483 9.098a.704.704 0 0 0-1.404.087l.704 11.344a.703.703 0 1 0 1.403-.087ZM12 8.438a.703.703 0 0 0-.703.703v11.343a.703.703 0 0 0 1.406 0V9.141A.703.703 0 0 0 12 8.437ZM16.262 8.439a.704.704 0 0 0-.745.658l-.703 11.344a.703.703 0 0 0 1.403.087l.704-11.344a.703.703 0 0 0-.659-.745Z"/>
-</svg>
\ No newline at end of file
+
+<svg viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M16.0511 3.87467H13.5396V3.40667C13.5396 2.63067 12.8639 2 12.0339 2H8.01535C7.18535 2 6.50963 2.63067 6.50963 3.40667V3.87467H3.99678C3.63989 3.87727 3.29555 3.99789 3.02495 4.21508C2.75436 4.43227 2.57502 4.73196 2.51882 5.06091C2.46262 5.38986 2.5332 5.72675 2.71801 6.01171C2.90282 6.29668 3.18989 6.51126 3.5282 6.61733L4.42392 16.7107C4.457 17.0613 4.6291 17.3878 4.90654 17.6263C5.18397 17.8648 5.54676 17.998 5.92392 18H14.1211C14.8982 18 15.5582 17.4333 15.6225 16.7107L16.5182 6.61733C16.8565 6.51126 17.1436 6.29668 17.3284 6.01171C17.5132 5.72675 17.5838 5.38986 17.5276 5.06091C17.4714 4.73196 17.2921 4.43227 17.0215 4.21508C16.7509 3.99789 16.4079 3.87727 16.0511 3.87467ZM7.51392 3.40667C7.51392 3.148 7.7382 2.93733 8.01535 2.93733H12.0339C12.3111 2.93733 12.5353 3.148 12.5353 3.40667V3.87467H7.51392V3.40667ZM14.6225 16.6333C14.6111 16.7501 14.5537 16.8587 14.4612 16.938C14.3688 17.0174 14.2481 17.0618 14.1225 17.0627H5.92678C5.8012 17.0618 5.68044 17.0174 5.58802 16.938C5.4956 16.8587 5.43812 16.7501 5.42678 16.6333L4.54392 6.68667H15.5053L14.6225 16.6333ZM16.0511 5.75067H3.9982C3.86484 5.75067 3.73693 5.70122 3.64263 5.6132C3.54833 5.52519 3.49535 5.40581 3.49535 5.28133C3.49535 5.15686 3.54833 5.03748 3.64263 4.94946C3.73693 4.86145 3.86484 4.812 3.9982 4.812H16.0511C16.1844 4.812 16.3123 4.86145 16.4066 4.94946C16.5009 5.03748 16.5539 5.15686 16.5539 5.28133C16.5539 5.40581 16.5009 5.52519 16.4066 5.6132C16.3123 5.70122 16.1844 5.75067 16.0511 5.75067Z" fill="#757575"/>
```

Comment:
Is hardcoding the fill color necessary here? We were using `currentColor` before. Can you please check?